LICENSE-3rdparty.csv

```diff
@@ -1,1408 +1,1408 @@
+core,github.com/hashicorp/hcl/json/token,MPL-2.0,UNKNOWN
+core,github.com/hashicorp/serf/coordinate,MPL-2.0,UNKNOWN
+core,github.com/hectane/go-acl,MIT,Nathan Osman
+core,github.com/hectane/go-acl/api,MIT,Nathan Osman
+core,github.com/huandu/xstrings,MIT,Huan Du
+core,github.com/ianlancetaylor/cgosymbolizer,BSD-3-Clause,"Free Software Foundation, Inc | The Go Authors"
+core,github.com/imdario/mergo,BSD-3-Clause,Dario Castañé | The Go Authors
+core,github.com/inconshreveable/mousetrap,Apache-2.0,Alan Shreve
+core,github.com/iovisor/gobpf/pkg/cpupossible,Apache-2.0,Kinvolk | PLUMgrid
+core,github.com/iovisor/gobpf/pkg/cpurange,Apache-2.0,Kinvolk | PLUMgrid
+core,github.com/itchyny/gojq,MIT,itchyny
+core,github.com/itchyny/timefmt-go,MIT,itchyny
+core,github.com/jlaffaye/ftp,ISC,Julien Laffaye <jlaffaye@FreeBSD.org>
+core,github.com/jmespath/go-jmespath,Apache-2.0,James Saryerwinnie
+core,github.com/josharian/intern,MIT,UNKNOWN
+core,github.com/josharian/native,MIT,Josh Bleecher Snyder
+core,github.com/json-iterator/go,MIT,json-iterator
+core,github.com/jstemmer/go-junit-report,MIT,Joel Stemmer
+core,github.com/jstemmer/go-junit-report/formatter,MIT,Joel Stemmer
+core,github.com/jstemmer/go-junit-report/parser,MIT,Joel Stemmer
+core,github.com/kardianos/osext,BSD-3-Clause,The Go Authors
+core,github.com/kjk/lzma,BSD-3-Clause,Andrei Vieru
+core,github.com/klauspost/compress/flate,BSD-3-Clause,Klaus Post | The Go Authors
+core,github.com/klauspost/compress/fse,BSD-3-Clause,Klaus Post | The Go Authors
+core,github.com/klauspost/compress/gzip,BSD-3-Clause,Klaus Post | The Go Authors
+core,github.com/klauspost/compress/huff0,BSD-3-Clause,Klaus Post | The Go Authors
+core,github.com/klauspost/compress/snappy,BSD-3-Clause,The Snappy-Go Authors
+core,github.com/klauspost/compress/zstd,BSD-3-Clause,Caleb Spare | Klaus Post | The Go Authors
+core,github.com/klauspost/compress/zstd/internal/xxhash,MIT,Caleb Spare | Klaus Post | The Go Authors
+core,github.com/klauspost/pgzip,MIT,Klaus Post
+core,github.com/kubernetes-sigs/custom-metrics-apiserver/pkg/apiserver,Apache-2.0,UNKNOWN
+core,github.com/kubernetes-sigs/custom-metrics-apiserver/pkg/apiserver/endpoints/handlers,Apache-2.0,UNKNOWN
+core,github.com/kubernetes-sigs/custom-metrics-apiserver/pkg/apiserver/installer,Apache-2.0,UNKNOWN
+core,github.com/kubernetes-sigs/custom-metrics-apiserver/pkg/apiserver/registry/rest,Apache-2.0,UNKNOWN
+core,github.com/kubernetes-sigs/custom-metrics-apiserver/pkg/cmd,Apache-2.0,UNKNOWN
+core,github.com/kubernetes-sigs/custom-metrics-apiserver/pkg/cmd/server,Apache-2.0,UNKNOWN
+core,github.com/kubernetes-sigs/custom-metrics-apiserver/pkg/dynamicmapper,Apache-2.0,UNKNOWN
+core,github.com/kubernetes-sigs/custom-metrics-apiserver/pkg/provider,Apache-2.0,UNKNOWN
+core,github.com/kubernetes-sigs/custom-metrics-apiserver/pkg/registry/custom_metrics,Apache-2.0,UNKNOWN
+core,github.com/kubernetes-sigs/custom-metrics-apiserver/pkg/registry/external_metrics,Apache-2.0,UNKNOWN
```

Comment:
all kubernetes-sigs/custom-metrics-apiserver sub folders don't have an authors file, top level does have an 'owners' file though, don't know if that's relevant. https://github.com/kubernetes-sigs/custom-metrics-apiserver/blob/master/OWNERS

Reply:
This is likely 'the Kubernetes Community'